manual/src/server/sal.md

```diff
@@ -0,0 +1,789 @@
+# SAL
+
+## Tfgrid
+
+### Load
+- loads key
+- chooses network
+- starts session and creates state for client
+
+```
+func (c *Client) Load(args Load) error
+```
+
+### DeployVM
+- Deploy a single VM 
+- Choose node id, farm id, etc 
+
+```
+func (c *Client) DeployVM(args tfgridBase.VMDeployment) (tfgridBase.VMDeployment, error)
+```
+
+### GetVMDeployment
+- Retrieve information about the deployed vm
+
+```
+func (c *Client) GetVMDeployment(name string) (tfgridBase.VMDeployment, error)
+```
+
+### CancelVMDeployment
+- Cancel the vm deployment
+
+```
+func (c *Client) CancelVMDeployment(name string) error
+```
+
+### DeployNetwork
```

Comment:
Not sure I understand this part, network is the private network (wireguard), but its details don't have flists, or vm specs and doesn't deploy vms

Reply:
So this function will be the function we already have [here](https://github.com/threefoldtech/web3_proxy/blob/d076cddde8ff1ca2df20b61d216ce7439fc9bd7d/server/clients/tfgrid/machines.go#L134) which does deploy vms that are on the same network. I would rename it to DeployNetwork, I think it makes more sense. The args object will be the object shown here: https://github.com/threefoldtech/web3_proxy/blob/d076cddde8ff1ca2df20b61d216ce7439fc9bd7d/server/clients/tfgrid/machines.go#L18
If you look into that object you will find the machines where you can choose the flists, VM specs, etc. I will probably rename some properties to more suitable names.